sashimi-webapp/src/components/Editor.vue

```diff
@@ -60,9 +73,14 @@ export default {
 <style scoped lang="scss">
 
   .viewer {
-    height: calc(100vh - 100px);
+    height: calc(100vh - 67px);
```

Comment:
Editor and viewer both uses `height: calc(100vh - 67px);`

Consider storing `67px` as a sass variable